agent/util-scripts/gold/pbench-clear-tools/test-65.txt

```diff
@@ -1,9 +1,9 @@
 +++ Running test-65 pbench-clear-tools --group=good
-Removed "vmstat" from host, "fubar", in tools group, "good"
 Removed "iostat" from host, "fubar", in tools group, "good"
+Removed "vmstat" from host, "fubar", in tools group, "good"
```

Comment:
Should we worry about the order?

Reply:
This should be fine

Reply:
I'm inclined to agree that this sort of minor ordering change, given that there doesn't appear to have been a particularly meaningful order before, probably isn't an issue. I just don't like the idea of gold files bouncing back and forth as we make code changes; it's sloppy. Then again, I don't like the whole idea of "gold file" unit testing in the first place, and now that you're building a Python command library, we can start moving towards something better.